updater/lib/dependabot/dependency_group_engine.rb

```diff
@@ -32,7 +32,7 @@ def self.from_job_config(job:) # rubocop:disable Metrics/PerceivedComplexity
         # - This is a security update and there are multiple dependencies passed in
         # Since there are no groups, the default behavior is to group all dependencies, so create a fake group.
         job.dependency_groups << {
-          "name" => "#{job.package_manager} group",
```

Comment:
@abdulapopoola this is where the second group was coming from

Reply:
the line above does add a new "group" entry though;